### DISCUSSION TITLE
Why 13.4.0 as current release but still 13.3.0 in the documentation?

### DISCUSSION THREAD

**Discussion by meyergru** [arendst/Tasmota#21006] [Q&A]
Is there a specific reason why the main documentation page on https://tasmota.github.io/docs/ still lists 13.3.0 as the current release while in fact that is apparently already at 13.4.0 for weeks now?

I almost missed the whole update because I use to look at the main page only...

**Comment by sfromis** [comment]
Updated, thanks :smile:
